tests/test_errorbundler.py

```diff
@@ -153,6 +156,8 @@ def test_states():
 
         if message["id"] == "comp":
             assert message["compatibility_type"] == "error"
+            assert message["editors_only"] == True
+            assert message["signing_severity"] == "high"
```

Comment:
I know you didn't add them yourself in this commit, but there's a couple of `print` statements in there. Would you mind removing them?